wandb/sdk/wandb_settings.py

```diff
@@ -415,7 +415,6 @@ class Settings:
     disabled: bool  # Alias for mode=dryrun, not supported yet
     docker: str
     email: str
-    enable_job_creation: bool
```

Comment:
do we have users that set this setting? do we need to let them know we removed it?

Reply:
Only private beta users are using this setting. I will communicate it to them to remove it